pkg/graph/schema.graphql

```diff
@@ -1445,6 +1445,27 @@ type UpdateSystemIntakePayload {
   userErrors: [UserError!]
 }
 
+"""
+Input for creating a Request Edits Action in Admin Actions v2
+"""
+input SystemIntakeRequestEditsInput {
+  systemIntakeID: UUID!
+  intakeFormStep: SystemIntakeFormStep!
+  notificationRecipients: EmailNotificationRecipients
+  emailFeedback: String!
+  additionalNotes: String
+  adminNotes: String
+}
+
+"""
+SystemIntakeRequestEditsOptions represents the current step in the intake process
+"""
+enum SystemIntakeFormStep {
```

Comment:
NIT: maybe this should be named to show it's specific to the request edits action? Something like `SystemIntakeRequestEditsFormStep`?

Reply:
I envisioned the form steps being reusable for other mutations and not specific to the request edits action when writing this, but if you think it should be more narrowly scoped in the name I can change it.

Reply:
I think it's probably fine. The fact that these three are the only actual `forms` makes the name specific enough. I originally was thinking that these were just a subset of the possible tasks (instead of identifying the forms specifically)